src/nspawn/nspawn.c

```diff
@@ -101,6 +104,11 @@
 #include "user-util.h"
 #include "util.h"
 
+/* Note that devpts's gid= parameter parses GIDs as signed values, hence we stay away from the upper half of the 32bit
+ * UID range here */
+#define UID_SHIFT_PICK_MIN ((uid_t) UINT32_C(0x00080000))
+#define UID_SHIFT_PICK_MAX ((uid_t) UINT32_C(0x6FFF0000))
```

Comment:
No, I wanted to leave some room at the beginning of the usable range, and some at the end of the usable range. Hence 0x7FFF0000 is not the right choice. (One might of course argue why I left 0x10000000 free at the end of the usable range, but only 0x80000 at the beginning of it, but I figured, there might be other stuff that might have longer ranges exclusively... i.e. at the beginning of the range there would only be individual allocations, while at the end there might be full ranges allocated...